### DISCUSSION TITLE
Settings for not editing the photos

### DISCUSSION THREAD

**Discussion by Pheggas** [immich-app/immich#2484] [Q&A] [answered]
Hello. I'd like to ask what settings should i tweak in order to achieve that Immich will not edit the photos by any means? I'd still like to use face recognition feature and others as well but i don't want to have these media edited by Immich.

So don't downscale images / videos, don't edit the name of those files, just purely copy new photos that aren't on server and process them for thumbnails and recognition.

**Comment by etnoy** [comment]
Not exactly sure what you want. Immich already stores original, unedited photos on the file system. Metadata above that is handled through the db

**Reply by Pheggas** [reply]
I noticed FFMpeg settings in Administration where i could transcode videos (and possibly images as well?) and also noticed the Storage Template but if i understand it the right way now, it only affects folder structure and not the files as they are. Am i correct?

**Reply by jrasm91** [reply, marked as answer]
Immichs generates a lot of content so that photos and videos can be played on different devices. All the generated content (including transcoded videos) are stored _in addition to_ the originals.

**Reply by jrasm91** [reply]
And yes the storage template moves _where_ originals are stored, but does not modify them.
